public_html/_footer.php

```diff
@@ -1,3 +1,36 @@
+<div id="ad_blocker_notice" style="display:none">
+  <h3>Looks like you are using an ad blocker!</h3>
+  <br>
+  Pageviews Analysis shows no ads, but the ad blockers blacklist websites that collect metrics.
+  In our case, we're just trying to show you metrics, not collect them!
+  <h2>
+    Please whitelist
+    <code>
+      tools.wmflabs.org
+    </code>
+  </h2>
+  <h4>AdBlock Plus</h4>
+  <p>
+    Click on the AdBlock Plus icon and select <i>Disable on tools.wmflabs.org</i>. Reload the page.
+  </p>
+  <h4>uBlock</h4>
+  <p>
+    Click on the uBlock icon and then click on the large power icon. Reload the page.
+  </p>
+</div>
+<div id="cookie_disabled_notice" class="col-lg-10 text-center" style="display:none">
+  <h3>Looks like you have cookies disabled!</h3>
+  <br>
+  <p>
+    Pageviews Analysis uses an internationalization library that requires to cookies to function.
```

Comment:
ha, thank you! I'm going to kill this notice anyway, assuming we can successfully get the app to function without cookies